### PR TITLE
Example: search-replace

### PR DESCRIPTION
**Pull request by phh**
#2769

**Search/replace production to development url (multisite compatible)**

Example code/script I'm using every day to search-replace from production to locally, regardless of it being a multisite or not.

### CODE REVIEW

File: php/commands/search-replace.php [hunk 1]
```diff
@@ -91,6 +91,13 @@ class Search_Replace_Command extends WP_CLI_Command {
 	 *
 	 *     # Search/replace to a SQL file without transforming the database
 	 *     wp search-replace foo bar --export=database.sql
+	 * 
+	 *     # Search/replace production to development url (multisite compatible)
+	 *     if $(wp --url=http://example.com core is-installed --network); then
```
Comment (danielbachhuber) [inline php/commands/search-replace.php]: What do you think about having some indication that this is a bash script meant to be run?

Comment (phh) [inline php/commands/search-replace.php]: Hmm. Maybe I could update the command to **Bash script: Search/replace production to development url (multisite compatible)** or add a `#!/bin/bash` at the start of it.

Any ideas?

Comment (danielbachhuber) [inline php/commands/search-replace.php]: > Maybe I could update the command to Bash script: Search/replace production to development url (multisite compatible) or add a #!/bin/bash at the start of it.

Yep, let's do both of those.

Comment (phh) [inline php/commands/search-replace.php]: Awesome! Both added.

**Pull request merged**